diff --git a/dist/vite/server.js b/dist/vite/server.js
index 0af59ac1f58a94a75d69c90a65a9070faf9cc2b6..7e1c87659a407fc72c342824c7e32a98835c60ec 100644
--- a/dist/vite/server.js
+++ b/dist/vite/server.js
@@ -177,6 +177,6 @@ async function createServer({ template, port, }) {
             throw e;
         }
     });
-    await app.listen({ port: port });
+    await app.listen({ port: port, listen: '0.0.0.0' });
     console.log(`Listening on: http://localhost:${port}`);
 }
